src/test/java/BaseTest.java

```diff
@@ -1,11 +1,175 @@
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));//5 seconds or 10 seconds?
+        driver.manage().window().maximize();
+        driver.navigate().to(baseURL);
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
+    }
+   /* @BeforeMethod
+    public void setupPage() {
+        String url = "https://qa.koel.app/";
+        driver.get(url);
+        Assert.assertEquals(driver.getCurrentUrl(), url);
+    }*/
+
+    @AfterMethod
+    public void tearDown() {
+        //threadDriver.get().close();// Parallel execution
+        //threadDriver.remove();
+        driver.quit(); //This method is applicable to sequential execution
+    }
+
+    public WebDriver lambdaTest() throws MalformedURLException{
+
+        String hubURL = "https://hub.lambdatest.com/wd/hub";
+
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+        ChromeOptions browserOptions = new ChromeOptions();
+        browserOptions.setPlatformName("Windows 11");
+        browserOptions.setBrowserVersion("121.0");
+        HashMap<String, Object> ltOptions = new HashMap<>();
+        ltOptions.put("username", "nayana.rao.subramanya");
+        ltOptions.put("accessKey", "jpX7iz5JkRTDxUMAICAibQPdZDTaxNpchbGfA7epSVtAfVYy9c");
+        ltOptions.put("build", "Selenium Grid Cloud");
+        ltOptions.put("project", "Selenium Grid Cloud-25");
+        ltOptions.put("selenium_version", "4.17.0");
+        ltOptions.put("w3c", true);
+        browserOptions.setCapability("LT:Options", ltOptions);
+
+        return new RemoteWebDriver(new URL(hubURL),browserOptions);
+    }
+/*
+    public void enterEmail(){
+        WebElement emailField = driver.findElement(By.cssSelector("[type=\"email\"]"));
+        emailField.sendKeys("nayana.rao.subramanya@testpro.io");
+    }
+
+    public void enterPassword(){
+        WebElement passwordField = driver.findElement(By.cssSelector("[type=\"password\"]"));
+        passwordField.sendKeys("Zqmvyk4hDaZ3vga");
+    }
+
+    public void loginButton() throws InterruptedException {
+        WebElement logInButton = driver.findElement(By.cssSelector("[type=\"submit\"]"));
+        logInButton.click();
+        Thread.sleep(1000);
+        WebElement avatardisplayed = driver.findElement(By.cssSelector("[class=\"avatar\"]"));
+        Assert.assertTrue(avatardisplayed.isDisplayed(), "Login Unsuccessful");
+    }
+
+    public void searchFieldAccess(){
+        WebElement searchField = driver.findElement(By.cssSelector("[type=\"search\"]"));
+        searchField.clear();
+        searchField.sendKeys("Tunnel of Lights (ID 1689)");
+    }
+
+    public void searchFieldAccess1(){
+        WebElement searchField = driver.findElement(By.cssSelector("[type=\"search\"]"));
+        searchField.clear();
+        searchField.sendKeys("Dee");
+    }
+
+    public void goHome(){
+        WebElement homePage = driver.findElement(By.cssSelector("[href=\"#!/home\"]"));
+        homePage.click();
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.